src/acachecontrol/cache.py

```diff
@@ -31,37 +38,36 @@ def __init__(self, config={}):
             "cacheable_methods", ("HEAD", "GET")
```

Comment:
Let's define constant for cacheable methods to be consistent.

Reply:
Done